docs/wiki/01_STUDY_PARAMETERS_SETUP/03_Loris_Modules/README.md

```diff
@@ -1,6 +1,13 @@
-List and describe each module that requires setting up or offers the possibility of customizing.
+# About Modules
+Each module is documented by a Readme in the codebase, under its specific modules/ subdirectory -- for example [here](https://github.com/aces/Loris/blob/main/modules/candidate_list/README.md).
 
-- statistics
-- candidate parameters
-- notifications
-- ...
\ No newline at end of file
+Setup instructions for each module are covered in its Readme.
+
+For a complete list of modules, see the directory [here](https://github.com/aces/Loris/blob/main/modules/).
+
+## How to make a LORIS module
+
+LORIS provides module manager to activate/deactivate a module for a whole study. The Creating a Module guide shall be found [here](https://github.com/aces/Loris/blob/main/docs/wiki/99_Developers/Creating_Module.md).
+
+## Customizing Loris modules
+For notes on how to customize modules, please consult the [Code Customization page](https://github.com/aces/Loris/wiki/Code-Customization#module-override)
```

Comment:
section out of place. As it clearly stated in the text that was erased and replaced with this... this section is to configure modules to be "enabled" or function normally. Not override them or make new ones which are developer tasks